scripts/extractor.cjs

```diff
@@ -1,126 +0,0 @@
-const parser = require("luaparse");
```

Comment:
Should we also remove the luaparse dependency now in package.json?

Reply:
Done